sport/test/football/feed/CompetitionsTest.scala

```diff
@@ -0,0 +1,80 @@
+package football.feed
+
+import java.time.{Clock, Duration, ZoneId, ZonedDateTime}
+
+import feed.Competitions
+import org.scalatest._
+import pa.FootballMatch
+import test.FootballTestData.{competitions, result, liveMatch, fixture}
+
+class CompetitionsTest extends FreeSpec with Matchers with OptionValues {
+  "Competitions" - {
+    Seq(
+      (false, Duration.ofSeconds(1), true),
+      (false, Duration.ofMinutes(1), true),
+      (false, Duration.ofMinutes(14).plusSeconds(59), true),
+      (false, Duration.ofMinutes(15), false),
+      (false, Duration.ofMinutes(16), false),
+      (true, Duration.ofMinutes(16), true),
+      (true, Duration.ofMinutes(20), true),
+    ) foreach { testcase =>
+      (s"isMatchLiveOrAboutToStart returns ${testcase._3} if there's a match that started ${testcase._2} ago") in {
+        val matches: Seq[FootballMatch] =
+          matchesWithLiveMatchAtCurrentMinusDuration(testcase._2, testcase._1)
+
+        val testCompetition = competitions(1).copy(matches = matches)
+        val competitionsList = Competitions(Seq(testCompetition))
+
+        val isMatchLiveOrAboutToStart = competitionsList.isMatchLiveOrAboutToStart(competitionsList.matches, clock)
```

Comment:
The trouble with using `isMatchLiveOrAboutToStart` here (and in the other tests below) is that it's also the name of the method we're testing! This means either this is a little confusing, and/or (if this were a function rather than a method) it makes importing the function a little more inconvenient.

I often use the name `result` to hold test output data that I'm about to perform an assertion on in situations like this, but whatever suits you!

Reply:
Yea I see your point. changed it to `result`